### PR TITLE
Dism module update for WS2022

### PR DESCRIPTION
**Pull request by PatAltimore**

### CODE REVIEW

**Comment by PatAltimore** [comment]
Hi @tiburd,

This one is ready. I just need two more approvers. Can you help?

Thanks,
Pat

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by PatAltimore** [comment]
Hi @tiburd,

Thanks for the help. This now has 3 approvals and is ready to merge.

Thanks,
Pat

**Review by dariomws — APPROVED** [review]

**Comment by tiburd** [comment]
@PatAltimore 
Hi Pat,

To clarify, Johan is a copy editor. 

To merge PowerShell PRs, they must be approved by Andres, Dario, and Trevor. 

So, when Johan has reviewed a PR, there are 4 approvals instead of only 3. 

Thanks!

**Pull request merged**
